### PR TITLE
Deprecate Athena and remove Chronicle; update get questions response

### PR DESCRIPTION
**Pull request by aliciaxw**
tested locally

response for getting all questions by group matches #167

### CODE REVIEW

File: src/repos/UsersRepo.js [hunk 1]
```diff
@@ -59,7 +59,9 @@ const createUserWithFields = async (googleID: string, firstName: string,
         await db().persist(user);
         return user;
     } catch (e) {
-        throw LogUtils.logError('Problem creating user!');
+        throw LogUtils.logErr(e, {
+            googleID, firstName, lastName, email,
+        }, 'Problem creating user!');
```
Comment (yuna878) [inline src/repos/UsersRepo.js]: Differentiate error message from `createUser` function so it's easier to distinguish which function is causing the error?

File: src/routers/v2/Groups/GetAdminGroupsRouter.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ class GetGroupsRouter extends AppDevRouter<Object> {
 
     async content(req: Request) {
         const groups = await UsersRepo.getGroupsByID(req.user.id, 'admin');
-        if (!groups) throw LogUtils.logError('Can\'t find groups for user!');
+        if (!groups) throw LogUtils.logErr({ message: 'Can\'t find groups for user!' });
```
Comment (yuna878) [inline src/routers/v2/Groups/GetAdminGroupsRouter.js]: Message should be more specific - `Can't find admin groups ...`

File: src/routers/v2/Groups/GetGroupsRouter.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ class GetGroupsRouter extends AppDevRouter<Object> {
 
     async content(req: Request) {
         const groups = await UsersRepo.getGroupsByID(req.user.id, 'member');
-        if (!groups) throw LogUtils.logError('Can\'t find groups for user!');
+        if (!groups) throw LogUtils.logErr({ message: 'Can\'t find groups for user!' });
```
Comment (yuna878) [inline src/routers/v2/Groups/GetGroupsRouter.js]: Same here - `Can't find member groups ...`

File: src/repos/GroupsRepo.js [hunk 4]
```diff
@@ -160,7 +160,7 @@ const addUsersByGoogleIDs = async (id: number, googleIDs: string[],
         await db().persist(group);
         return group;
     } catch (e) {
-        throw LogUtils.logError('Problem adding users to group by google ids!');
+        throw LogUtils.logErr(e, { id, googleIDs, role }, 'Problem adding users to group by google ids!');
```
[the remark this answers is not in the document]
Comment (meganle) [reply src/repos/GroupsRepo.js]: Picky is good! I agree with Yuna here

File: src/repos/GroupsRepo.js [hunk 5]
```diff
@@ -232,7 +232,7 @@ const removeUserByGroupID = async (id: number, user: User, role: ?string):
 
         return group;
     } catch (e) {
-        throw LogUtils.logError(`Problem removing admin from group by id: ${id}`);
+        throw LogUtils.logErr(e, { user, role }, `Problem removing admin from group by id: ${id}`);
```
[the remark this answers is not in the document]
Comment (meganle) [reply src/repos/GroupsRepo.js]: I think specific is good, since we'll be able to get more details

[the remark this answers is not in the document]
Comment (aliciaxw) [reply src/repos/GroupsRepo.js]: alternatively i could have the message be general and put id into the data: `{ id, user, role }`, thoughts?

[the remark this answers is not in the document]
Comment (yuna878) [reply src/repos/GroupsRepo.js]: Doesn't this error message apply to both admins and members though? Isn't that why `role` is being passed in?

[the remark this answers is not in the document]
Comment (aliciaxw) [reply src/repos/GroupsRepo.js]: oh disregard previous msg i totally misread what you said

[the remark this answers is not in the document]
Comment (aliciaxw) [reply src/repos/GroupsRepo.js]: you're right, the function is for all users so the message should be generalized

**Comment by aliciaxw** [comment]
addressed PR suggestions

**Pull request merged**